dist/stat_data.py

```diff
@@ -243,6 +243,7 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_eviction_aggressive_set', 'eviction currently operating in aggressive mode', 'no_clear,no_scale'),
     CacheStat('cache_eviction_app', 'pages evicted by application threads'),
     CacheStat('cache_eviction_app_dirty', 'modified pages evicted by application threads'),
+    CacheStat('cache_eviction_clear_page_from_non_urgent_queues', 'Page cleared from non-urgent eviction queue'),
```

Comment:
Here is a suggested rewording:

`CacheStat('cache_eviction_clear_ordinary', 'pages removed from the ordinary queue to be queued for urgent eviction'),`

I would like the words "queued for urgent eviction" to be included in the description, as all urgent eviction stats use those words, and it helps me search through ftdc/T2 more intuitively.

Reply:
Sounds good, updated in [69b0526](https://github.com/wiredtiger/wiredtiger/pull/8950/commits/69b05267a51edf9ab07a0710b98bb60a7ac53a62)